lib/icedfrisby.js

```diff
@@ -1159,6 +1144,8 @@ class Frisby {
         timeout: this._timeout,
       }, this.current.request, params)
```

Comment:
This means that in the following circumstance, addHeader isn't respected. This doesn't look like new behaviour in this PR. I've got a unit test for this, but don't wanna commit it if we don't wanna fix it now.

```
frisby.create(this.test.title)
      .baseUri('http://httpbin.org')
      .get('/headers', {headers:{'a':'1', 'b':'1'}})
      .addHeader('a','2')
.toss()
```

Reply:
I've pushed the unit test as an `it.skip` for now